lib/nerves_runtime.ex

```diff
@@ -3,6 +3,7 @@ defmodule Nerves.Runtime do
 
   alias Nerves.Runtime.OutputLogger
 
+  @target System.get_env("MIX_TARGET")
```

Comment:
can we use `Mix.Project.config()[:target]` here?

Reply:
Nope, when used as a path dependency, `Mix.Target.config()[:target]` would reference the `Mix.Project` for `:nerves_runtime` instead of the project at the top of the stack.